WordPress/Classes/Utility/InteractiveNotificationsManager.swift

```diff
@@ -502,4 +508,8 @@ extension InteractiveNotificationsManager: UNUserNotificationCenterDelegate {
             completionHandler()
         }
     }
+
+    func userNotificationCenter(_ center: UNUserNotificationCenter, openSettingsFor notification: UNNotification?) {
+        MeNavigationAction.notificationSettings.perform([:])
```

Comment:
I saw `func perform(_ values: [String: String], source: UIViewController? = nil)` doesn't use the value at the moment. What do you think if we remove it or add `[:]` as default value in order to use `.perform()` only?

Reply:
This actually used to be optional, but @diegoreymendez changed it recently: https://github.com/wordpress-mobile/WordPress-iOS/pull/11851/files

While we can't add a default parameter to the protocol (I think?) I just added one to MeNavigationAction.